### DISCUSSION TITLE
Is there a link check limit? Have I found it? 😄

### DISCUSSION THREAD

**Discussion by JayCuthrell** [buttondown/discussions#10] [Q&A]
In my most recent newsletter issue [#194 — Take the Bot DataDome](https://fudge.org/archive/take-the-bot-datadome/), the Send modal popup presented this error:

<img width="329" alt="Screenshot 2023-04-09 at 8 12 00 PM" src="https://user-images.githubusercontent.com/95317/230803491-377ffe80-2aa8-45df-ba10-836213115a44.png">

**Comment by jmduke** [comment]
We fixed this one 😅 .
